flatten-array.json

```diff
@@ -0,0 +1,29 @@
+{
+   "cases": [
+      {
+         "description": "flattens array with null and integers present",
+         "input": [[]],
+         "expected": []
+      },
+      {
+         "description": "flattens array with just integers present",
+         "input": [1,[2,3,4],5],
+         "expected": [1,2,3,4,5]
+      },
+      {
+         "description": "flattens array with just null",
+         "input": [1,[2,3,4],5,[6,[7,8]]],
+         "expected":[1, 2, 3, 4, 5, 6, 7, 8]
+      },
+      {
+         "description": "flattens array with just null",
```

Comment:
This has the same description as the previous one. In Ruby if you have several tests with the same name, then only the last one will be run.

Also, this doesn't have nulls in it.